service/service.go

```diff
@@ -95,6 +96,27 @@ func New(config Config) (*Service, error) {
 		}
 	}
 
+	var bootstrapRunner *bootstrap.Bootstrap
+	{
+		c := bootstrap.Config{
+			Logger:    config.Logger,
+			K8sClient: k8sClient,
+
+			CustomerAdminGroup: config.Viper.GetString(config.Flag.Service.NamespaceAuth.WriteAllCustomerGroup),
+			GSAdminGroup:       config.Viper.GetString(config.Flag.Service.NamespaceAuth.WriteAllGSGroup),
+		}
+
+		bootstrapRunner, err = bootstrap.New(c)
+		if err != nil {
+			return nil, microerror.Mask(err)
+		}
+	}
+
+	err = bootstrapRunner.Run()
```

Comment:
What do you think about moving this to the `Boot` method? I feel like `New` functions should only configure objects and not do any heavy lifting.

Reply:
moved